src/app/js/theme.js

```diff
@@ -1,11 +1,5 @@
 export default {
-    green: {
-        primary: '#7DBD42',
-    },
-    purple: {
-        primary: '#B22F90',
-    },
-    orange: {
-        primary: '#F48022',
```

Comment:
I wrote `orange.secondary` in another PR ... Can you keep it here ? :/

Reply:
done